recovery/elliptics_recovery/iterator.py

```diff
@@ -322,6 +322,8 @@ def start(self,
             else:
                 yield results[0]
         except Exception as e:
+            if hasattr(e, "code"):
```

Comment:
exception.Exception class doesn't have attribute "code". I've added this attr only to elliptics.core.Error.